.travis.yml

```diff
@@ -21,11 +21,6 @@ env:
   matrix:
     - RUN_LINT=true
     - RUN_FRONTEND_TESTS=true
-    - RUN_BACKEND_TESTS=true REPORT_BACKEND_COVERAGE=false EXCLUDE_LOAD_TESTS=true
```

Comment:
You know ... I think we should actually put this in a comment rather than deleting it. Add a note to say it's being run on Circle-CI. It will enable easy switching if Circle-CI has issues.